platform/ios/src/MGLAnnotationView.h

```diff
@@ -141,6 +141,19 @@ typedef NS_ENUM(NSUInteger, MGLAnnotationViewDragState) {
  */
 @property (nonatomic, assign) BOOL scalesWithViewingDistance;
 
+/**
+ A Boolean value that determines whether the annotation view rotates together
+ with the mapview.
```

Comment:
Nit: replace “mapview” with “map”, since the `direction` property reflects the rotation of the content within the map view, not the map view itself.